libbpf-tools/user_map_helpers.c

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+// Copyright (c) 2020 Anton Protopopov
+#include <stdlib.h>
+#include <stdio.h>
+#include <errno.h>
+
+#include "user_map_helpers.h"
```

Comment:
with .bpf.h convention I mentioned above, this could be just `map_helpers.h`

Reply:
+1